Dockerfile

```diff
@@ -12,7 +12,14 @@ LABEL maintainer=terraform-linters
 
 RUN apk add --no-cache ca-certificates
 
-COPY --from=builder /tflint/dist/tflint /usr/local/bin
+# copy tflint from builder
+COPY --from=builder /tflint/dist/tflint /usr/local/bin/tflint
 
-ENTRYPOINT ["tflint"]
+# set working directory
 WORKDIR /data
+
+# set the default entrypoint -- when this container is run, use this command
+ENTRYPOINT [ "tflint" ]
+
+# as we specified an entrypoint, this is appended as an argument (i.e., `tflint --help`)
+CMD [ "--help" ]
```

Comment:
Is this really the best default command, as opposed to invoking TFLint with no arguments, which is implicitly the same as `tflint .`?

Reply:
reverting back to original behaviour